ui/selectors/nft.ts

```diff
@@ -1,21 +1,30 @@
-import { NftContract } from '@metamask/assets-controllers';
+import { Nft, NftContract } from '@metamask/assets-controllers';
 import { createSelector } from 'reselect';
 import { getMemoizedCurrentChainId } from './selectors';
 
-type NftState = {
+export type NftState = {
   metamask: {
     allNftContracts: {
       [account: string]: {
         [chainId: string]: NftContract[];
       };
     };
+    allNfts: {
+      [account: string]: {
+        [chainId: string]: Nft[];
+      };
+    };
   };
 };
 
 function getNftContractsByChainByAccount(state: NftState) {
   return state.metamask.allNftContracts ?? {};
 }
 
+function getNftsByChainByAccount(state: NftState) {
```

Comment:
Nit: JSDoc blocks for these new selectors would be appreciated

Reply:
Done.